src/pages/indice-culturel/success.tsx

```diff
@@ -0,0 +1,30 @@
+import Link from 'next/link';
```

Comment:
can be good to refactor the success page to only have one component for every activity